CONTRIBUTING.rst

```diff
@@ -54,6 +54,14 @@ Depending on where you want to make your contribution please choose your correct
 | RHEL-X / CentOS Stream X | rhel-x       |
 +--------------------------+--------------+
 
+Notice that the ``master`` branch should only be a target of a pull request if the change does not apply to any
+of the branched Fedora versions. If the change applies to both Fedora Rawhide and Fedora XX, then a ``fXX-devel``
+branch has to be targeted. All changes to ``fXX-devel`` branches are merged back into ``master``, so there's no
+need to open a separate pull-request for ``master`` in that case.
+
+The ``rhel-x`` branches on the other hand are not merged back into ``master``, and so a separate pull request
+has to be opened targeting either for a ``fXX-devel`` branch or ``master``.
```

Comment:
I'm not sure if that is within the scope here, but we use the labels "port to X" where X = Fedora or RHEL. Having that label on a PR means this should be in the other branch too, but the PR was not yet created.

Arguably, that is of little use to contributors because they can not add these labels.

Reply:
> Arguably, that is of little use to contributors because they can not add these labels.

Yeah, I always thought that this kind of labeling for PR was something handled by the Anaconda core developers as a part of the PR review process.